docs/product-migration-guide/src/main/asciidoc/migration-update-remote-api-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='migration-update-remote-api-proc']
+= Replace API references to {CENTRAL} execution server
+
+In {PRODUCT} {PRODUCT_VERSION}, {CENTRAL} no longer has execution server capabilities nor any API endpoints related to the execution server. Instead, the {KIE_SERVER} performs all execution and runtime functions and {CENTRAL} is used only for designing and managing decision services. Therefore, if any of your applications interact with the {CENTRAL} execution server through the remote Java API (`kie-remote-client`) or through direct REST API calls (`/business-central/rest/`), you must update these API references to redirect to {KIE_SERVER}.
+
+.Procedure
+. In your application code for remote Java clients, replace any instances of the following legacy package and module for {CENTRAL} with the new package and module for {KIE_SERVER}. This replaces references to the remote Java API with references to {KIE_SERVER}.
++
+--
+Legacy {CENTRAL} package and module:
+[source,java]
+----
+org.kie.remote.client
+kie-remote-client
+----
+
+New {KIE_SERVER} package and module:
+[source,java]
+----
+org.kie.server.client
+kie-server-client
+----
+--
+. Wherever applicable in your application code, replace any legacy REST API calls in the format `http://<SERVER>:<PORT>/business-central/rest/<ENDPOINT>` with new REST API calls in the format `http://<SERVER>:<PORT>/kie-server/services/rest/server<ENDPOINT>`. This replaces references to the {CENTRAL} execution server with references to {KIE_SERVER}.
```

Comment:
Missing a slash before the second `<ENDPOINT>`.

Reply:
Good catch! Done.